packages/google_sign_in/google_sign_in_web/lib/src/flexible_size_html_element_view.dart

```diff
@@ -78,6 +78,8 @@ class _FlexHtmlElementView extends State<FlexHtmlElementView> {
   ) {
     final web.DOMRectReadOnly rect = resizes.toDart.last.contentRect;
     if (rect.width > 0 && rect.height > 0) {
+      // The following ignore can be removed once the package leaves web:0.5.1 behind.
```

Comment:
This should be a TODO.

Reply:
Made a TODO, and linked to the cleanup issue.